src/test/java/reactor/kafka/receiver/KafkaReceiverTest.java

```diff
@@ -441,12 +440,12 @@ public void manualCommitSync() throws Exception {
         receiverOptions = receiverOptions.commitInterval(Duration.ZERO).commitBatchSize(0);
         KafkaReceiver<Integer, String> receiver = createReceiver();
         Flux<? extends ConsumerRecord<Integer, String>> kafkaFlux = receiver.receive()
-                         .doOnNext(record -> {
-                                 assertEquals(committedOffsets[record.partition()], record.offset());
-                                 record.receiverOffset().commit()
-                                       .doOnSuccess(i -> onCommit(record, commitLatch, committedOffsets))
-                                       .block();
-                             })
+                         .concatMap(record -> {
```

Comment:
`.delayUntil` to avoid `.then(Mono.just(record))`?

Reply:
done